toolchains/workspace-pnpm/macros.bzl

```diff
@@ -494,6 +494,12 @@ _next_build = rule(
             default = "toolchains//:workspace_pnpm",
             providers = [WorkspacePnpmToolchainInfo],
         ),
+        "prod_deps_srcs": attrs.dict(
```

Comment:
Adding this isn't going to have the desired effect. Also need to handle the argument in the implementation cc @vindard

Reply:
> Adding this isn't going to have the desired effect. Also need to handle the argument in the implementation cc @vindard

https://github.com/GaloyMoney/galoy/blob/3c150ca73d2a8a81277d58d50ef5e2e128272576/toolchains/workspace-pnpm/macros.bzl#L568
I think `prod_deps_srcs`  handles this, but yes, @vindard may have more context.